lib/lts/ember-lts.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
When I investigated it seemed that ember's LTS is algorithmic, what did I miss, why do  we need to hard-code?

Reply:
For instance [3.20.0 released ](https://github.com/ember-cli/ember-cli/releases/tag/v3.20.0)on July 27. In the website, it says 3.20 LTS active from Aug 24. There is a month difference. So do math wrt what we get from npm registry we my call EOL earlier than actual.